src/content/guide/tools-and-features/device-updater.md

```diff
@@ -0,0 +1,100 @@
+---
+word: Updater
+title: Device Updater
+order: 4
+columns: two
+devices: [electron]
```

Comment:
No other Tools page is shown conditionally for only one device.

The consequence if that if you are viewing the docs for the Photon and click on a link in the Tools section, when the page loads you'll see a new link "Device Updater" show up in the TOC.

I would just remove the `devices: [electron]` for now.